### ISSUE TITLE
gpg4win-light.sls fails to install silently

### ISSUE THREAD

**Issue by TheBigBear** [saltstack/salt-winrepo#113]
Unfortunately the gpg4win-light sls https://github.com/saltstack/salt-winrepo/blob/master/gpg4win-light.sls fails to complete installing under salt, because the upstream installer has a bug and the /S cmd line switch does not work completely. It will still ask interactively: "Do you want Claws Mail your default mail client?", and because salt installations do not have a UI, it won't finish installing. Bug ir reported upstream, and I will fix the installer sls as sson as they fix it upstream.

**Comment by UtahDave** [comment]
That's great. I really appreciate all the work you've done to add packages and update others.
I'm about to catch a plane in Brussels heading back home to the States, but I'll finish processing them once I get back home.

**Comment by TheBigBear** [comment]
No problem, oh that's why you had such 'strange' (for a person from Utah US) merging hours? And I thought you got up in the middle of the night, to merge mine. LOL ;-) Have a save flight.

**Comment by TheBigBear** [comment]
Ok, there is a temporary 'local' work-around, until upstream have fixed the broken 2.2.1 "light" installer.
for now you can run the installer as:
```
gpg4win-light-2.2.1.exe /S /C=C:\temp\gpg4win.ini 
```
and it won't ask about the Claws Mail being default mailer anymore.

The gpg4win.ini file needs the following contents for that:
```
[gpg4win]
  ; Installation settings. Leave out or leave empty for
  ; Default settings
  inst_claws_mail = false
```
